### PR TITLE
K8s

### PR DESCRIPTION
**Pull request by awolfe76**
# This is to the `v2` branch.

### Depends on #164 

Adds Helm chart setup for Kubernetes.

### CODE REVIEW

File: kubernetes/hmda-platform-tools/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "hmda-pub-ui.name" -}}
```
Comment (wpears) [inline kubernetes/hmda-platform-tools/templates/_helpers.tpl]: pub-ui -> tools

File: kubernetes/hmda-platform-tools/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "hmda-pub-ui.fullname" -}}
```
Comment (wpears) [inline kubernetes/hmda-platform-tools/templates/_helpers.tpl]: pub -> tools

File: kubernetes/hmda-platform-tools/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "hmda-pub-ui.chart" -}}
```
Comment (wpears) [inline kubernetes/hmda-platform-tools/templates/_helpers.tpl]: pub-ui -> tools

**Comment by awolfe76** [comment]
Good catch @wpears. Fixed in 18edd23. 🔨

**Pull request merged**